Home.md

```diff
@@ -16,23 +16,23 @@ Fork the [wiki repository](https://github.com/LaunchCodeEducation/csharp-web-dev
-* [Class 12 - Model Validation and Enums](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Class-12-(Model-Validation-and-Enums))
-* [Class 13 - Work Day](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Class-13-(Work-Day))
-* [Class 14 - Databases and SQL](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Class-14-(SQL-Part-1))
-* [Class 15 - The SQL Sequel](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Class-15-(SQL-Part-2))
-* [Class 16 - Introduction to Object-Relational Mapping](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Class-16-(ORM-Part-1))
-* [Class 17 - Relationships in Object-Relational Mapping](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Class-17-(ORM-Part-2))
-* [Class 18 - ORM Part 3](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Class-18-(ORM-Part-3))
-* [Class 19 - Authentication](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Class-19-(Authentication))
-* [Class 20 - REST](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Class-20-(REST))
\ No newline at end of file
+* [Lesson 1 - Data & Variables](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-1-(Data-&-Variables))
+* [Lesson 2 - Control Flow & Collections](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-2-(ControlFlow-&-Collections))
+* [Lesson 3 - Classes & Objects](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-3-(Classes-&-Objects))
+* [Lesson 4 - Classes, Part 2](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-4-(Classes-Part-2))
+* [Lesson 5 - Unit Testing](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-5-(Unit-Testing))
+* [Lesson 6 - Inheritance](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-6-(Inheritance))
+* [Lesson 7 - Interfaces](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-7-(Interfaces))
+* [Lesson 8 - Intro to MVC and Controllers](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-8-(Intro-to-MVC-and-Controllers))
+* [Lesson 9 - Exceptions and Build Tools](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-9-(Exceptions-&-Build-Tools))
+* [Lesson 10 - Razor Views](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-10-(Razor-Views))
+* [Lesson 11 - Model Classes and Model Binding](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-11-(Model-Classes-and-Model-Binding))
+* [Lesson 12 - Model Validation and Enums](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-12-(Model-Validation-and-Enums))
+* [Lesson 13 - Work Day](https://github.com/LaunchCodeEducation/csharp-web-development/wiki/Lesson-13-(Work-Day))
```

Comment:
same here as w/ java - let's remove these instructions as a particular "lesson" and re-work them into a general info page?

Reply:
Done! Kept the lesson listing there, but it directs back to a general info page. General info is also found in the course overview pages.